docs/help/getting-started.md

```diff
@@ -1,27 +1,56 @@
 # Deploying Bump.sh
 
-Get started in a few minutes with Bump!
+Get started in a few minutes with Bump.sh!
 
 ## Create your first documentation
-Step 1: Create your space/documentation
-Step 2: Upload your file
-    - upload file directly
-    - use the CLI
-    - try an example
-Step 3: Enjoy
+
+### Step 1: Create your space/documentation
+
+After creating your account, you'll be asked to name your first API documentation.
```

Comment:
```suggestion
After creating your account, you'll be asked [to create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation.
```
How about adding those direct link anywhere possible (for instance also for Hubs and Orgs)